app/presenters/chf_index_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class ChfIndexPresenter < Blacklight::IndexPresenter
+  class_attribute :description_max_chars
+  self.description_max_chars = 10
+
+  attr_reader :description_field
+
+  def initialize(*args)
+    super
+    # This is an attempt to not hard-code in that our field is called
+    # `description_tesim`, so that doesn't break if it's changed, we
+    # just look for the "description" field. Not really sure if this is
+    # really less fragile, or if there was a better place to get this.
+    @description_field = configuration.index_fields.values.
+      find { |conf| conf.itemprop == "description" }.
```

Comment:
ah, okay! I see it there. thanks.

I was concerned that in the model a property may be defined with more than one index field; how do we know which index field we're getting? But looking at the CatalogController I see that's not an issue.